stl/inc/type_traits

```diff
@@ -1877,8 +1934,8 @@ public:
     }
 
     template <class... _Types>
-    auto operator()(_Types&&... _Args) const -> decltype(_STD invoke(get(), _STD forward<_Types>(_Args)...)) {
-        return _STD invoke(get(), _STD forward<_Types>(_Args)...);
+    auto operator()(_Types&&... _Args) const -> decltype(_STD invoke(get(), static_cast<_Types&&>(_Args)...)) {
+        return _STD invoke(get(), static_cast<_Types&&>(_Args)...);

```

Comment:
Pre-existing: `get()` returns `*_Ptr`. I think this class is simple enough that we can simply "punch through" and use `*_Ptr` in the function body and `decltype`.

Reply:
Casey pointed out that the problem was not putting _Ptr first, not anything related to this->, so did that reordering.